src/transformers/__init__.py

```diff
@@ -360,6 +360,12 @@
     "models.herbert": ["HerbertTokenizer"],
     "models.hubert": ["HUBERT_PRETRAINED_CONFIG_ARCHIVE_MAP", "HubertConfig"],
     "models.ibert": ["IBERT_PRETRAINED_CONFIG_ARCHIVE_MAP", "IBertConfig"],
+    "models.idefics": [
+        "IDEFICS_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "IdeficsConfig",
+        "IdeficsImageProcessor",
```

Comment:
This one requires vision according to the intermediate init.

Reply:
This was resolved but not addressed. The ImageProcessor is only available if `is_vision_available()` in the init of the model, so it should be the same here.